Doc/library/tempfile.rst

```diff
@@ -388,6 +391,7 @@ Here are some examples of typical usage of the :mod:`tempfile` module::
     >>> fp.write(b'Hello world!')
     # read data from file
     >>> fp.seek(0)
+    0
```

Comment:
If we add `0` here, we should add `12` after `fp.write(b'Hello world!')` too.

Reply:
And later,

```
    >>> with tempfile.TemporaryFile() as fp:
    ...     fp.write(b'Hello world!')
    ...     fp.seek(0)
    ...     fp.read()
    b'Hello world!'
```

This output should be:

```
12
0
b'Hello world!'
```

Reply:
It seems this file is not tested by doctest.

Please leave it. Otherwise, we need to fix all output and remove all comments in sample code.